theme/js/dynamic-scripts.js

```diff
@@ -8,19 +8,41 @@ $(document).ready(function() {
     getDonationSumData();
 });
 
+$.fn.extend({
+    toggleText: function(a, b){
+        return this.text(this.text() == b ? a : b);
+    }
+});
```

Comment:
Code comments on how this function works please.

Reply:
A resource on ternary operators: https://guide.freecodecamp.org/c/ternary-operator/

Reply:
Information on $.fn.extend: https://www.google.com/search?ei=ZDHDXaScJ9aH1fAPioyQyAg&q=jquery+extend+function+%24.fn&oq=jquery+extend+function+%24.fn&gs_l=psy-ab.3..0i71l8.0.0..10233...0.2..0.0.0.......0......gws-wiz.e8KBCXi3Mn0&ved=0ahUKEwik3rfDutblAhXWQxUIHQoGBIkQ4dUDCAs&uact=5

Reply:
Thank you.